_documentation/gitlab.md

```diff
@@ -1,11 +1,109 @@
 ---
 layout: page
 title:  "Testing in GitLab"
-cover-img: /assets/img/wip_splash.jpg
-date:   2024-04-19 20:00:14 +0100
+cover-img: /assets/img/infinite_splash.jpg
+date:   2024-06-23 22:16:15 +0100
 author: Dan Caseley
 ---
 
-Coming soon.
+## Introduction
 
-_Splash image courtesy of [Pavel Neznanov, Unsplash](https://unsplash.com/photos/silver-and-gold-round-accessory-w95Fb7EEcjE?utm_content=creditShareLink&utm_medium=referral&utm_source=unsplash)_
+This guide is intended for XMPP Server developers who are building in GitLab. It demonstrates how to add XMPP Interoperability and Standards Conformance tests integrated into a build & test pipeline. This assumes that you already have a pipeline established, and wish to add the additional checks.
+
+Like most external jobs in GitLab, this one consists of both an included component and an explicit to call it as a job. When called, it runs the tests, giving junit-esque outputs and debug logs. The only prerequisite is that you've got a built server, and have started it.
```

Comment:
This is a bit of a confusing sentence. Did you mean "Like most external jobs in GitLab, this one consists of both an included component and an explicit component to call it as a job."

Reply:
I think I improved it. How about now?